### ISSUE TITLE
Add-on missing permission for instrumenting wiso-net.de

### ISSUE THREAD

**Issue by Crazyphil** [stefanw/voebbot#41]
I'm using Firefox 87 with VÖBBot 0.16.0, configured with the WISO (SSO) provider.

When I open any page with paid content, the WISO tab automatically loads in the background. However, after that, nothing happens anymore. The add-on message on the news page stays at "Artikel wird gesucht...". According to the browser log, I first get the `tab created` debug message from the add-on, followed by `tab load complete`. After that, however, nothing else gets logged to the console, but a few lines after the last message, an error stating `Error: Missing host permission for the tab` appears. This message, however, has a source of "undefined", so I can't definitely say that it originates from the add-on's actions.

This got me curious, so I checked the add-on's permissions. It lists various news websites and genios.de, but there are no permissions requested for wiso-net.de, which should be instrumented by the add-on when searching for articles. The manifest of the add-on only lists this domain as an "optional permission" and it seems that my Firefox browser doesn't grant them (and I also don't have an idea how I would do that myself).

![grafik](https://user-images.githubusercontent.com/3602847/116197189-b3b42e00-a734-11eb-9c0b-30bc76b6e060.png)

**Comment by stefanw** [comment]
Permissions will now be updated when saving on the options page. Make sure you have version 0.17.0+ and click save again.

**Comment by Crazyphil** [comment]
Unfortunately, the problem persists with version 0.17.0. The browser console shows the same error as before, and wiwo-net.de is still not shown in the permissions. In addition to just clicking the Save button, I also uninstalled the add-on, restarted the browser and reinstalled it.
However, the settings seem to be stored even when the add-on is uninstalled, because when I opened the settings tab after reinstalling the add-on, everything was still configured as before the uninstallation.
